tinymce4-parent/tinymce4/src/main/java/wicket/contrib/tinymce4/settings/TinyMCESettings.java

```diff
@@ -415,10 +411,138 @@ private Mode(String name)
 	/**
 	 * Language enum
 	 */
-	public static enum Language
-	{
-		ar, bg, bs, ca, ch, cs, da, de, el, en, es, et, fa, fi, fr, gl, he, hr, hu, ia, ii, is, it, ja, ko, lt, lv, mk, ms, nb, nl, nn, pl, pt, ro, ru, sc, se, si, sk, sl, sr, sv, tr, tt, tw, uk, vi, zh
-	}
+	public enum Language
+	{
+        ar,
+        ar_SA,
+        az,
+        be,
+        bg_BG,
+        bn_BD,
+        bs,
+        ca,
+        cs,
+        cs_CZ,
+        cy,
+        da,
+        de,
+        de_AT,
+        dv,
+        el,
+        en_CA,
+        en_GB,
+        eo,
+        es,
+        es_MX,
+        et,
+        eu,
+        fa,
+        fa_IR,
+        fi,
+        fo,
+        fr_CH,
+        fr_FR,
+        ga,
+        gd,
+        gl,
+        he_IL,
+        hi_IN,
+        hr,
+        hu_HU,
+        hy,
+        id,
+        is_IS,
+        it,
+        ja,
+        ka_GE,
+        kab,
+        kk,
+        km_KH,
+        ko,
+        ko_KR,
+        ku,
+        ku_IQ,
+        lb,
+        lt,
+        lv,
+        mk_MK,
+        ml,
+        ml_IN,
+        mn_MN,
+        nb_NO,
+        nl,
+        pl,
+        pt_BR,
+        pt_PT,
+        ro,
+        ru,
+        ru_RU,
+        si_LK,
+        sk,
+        sl_SI,
+        sr,
+        sv_SE,
+        ta,
+        ta_IN,
+        tg,
+        th_TH,
+        tr,
+        tr_TR,
+        tt,
+        ug,
+        uk,
+        uk_UA,
+        vi,
+        vi_VN,
+        zh_CN,
+        zh_TW;
+
+        private static final Map<String, Language> unmappedLocales = new HashMap<>();
+
+        static {
+        	unmappedLocales.put("bg", Language.bg_BG);
+        	unmappedLocales.put("en", Language.en_GB);
+			unmappedLocales.put("fr", Language.fr_FR);
+			unmappedLocales.put("hi", Language.hi_IN);
+			unmappedLocales.put("hu", Language.hu_HU);
+			unmappedLocales.put("is", Language.is_IS);
+			unmappedLocales.put("iw", Language.he_IL);
```

Comment:
I've applied [wicket style settings](https://cwiki.apache.org/confluence/display/WICKET/Wicket+Source+Code+Style).
Hope that fixes it.